### DISCUSSION TITLE
Is this a buug with KDE or KeepassXC?

### DISCUSSION THREAD

**Discussion by SuperSentient** [keepassxreboot/keepassxc#8893] [Q&A] [answered]
Sorry for putting this here frankly I'm not even sure if this a bug or not.
For about a month or two now I noticed a bug with KeepassXC when I'm closing it doesn't always close right away in my panel.
Is this a bug with KDE Plasma  or KPXC?

**Comment by droidmonkey** [comment, marked as answer]
It is a kde problem that has been reported before. It only happens on kde.

**Reply by SuperSentient** [reply]
Is it harmful? Is it safe to save changes in it on KDE Plasma?

**Reply by droidmonkey** [reply]
I don't see why that would be harmful?

**Reply by SuperSentient** [reply]
I have my KPXC configured not to automatically save changes and when I do save I do it via closing the app and choosing to save changes before I close the app. Could that cause any problems?

**Reply by droidmonkey** [reply]
That is likely why it's delayed closing, saving locks the ui until its done and depending on your selection of encryption parameters that could take several seconds.

**Reply by SuperSentient** [reply]
But is that potentially dangerous for KDBX files?

**Reply by droidmonkey** [reply]
No

**Reply by SuperSentient** [reply]
Thanks. :+1:  :)
